src/components/AbortionButton/AbortionButton.stories.js

```diff
@@ -0,0 +1,78 @@
+import {expect} from '@storybook/jest';
+import {waitFor, within} from '@storybook/testing-library';
+
+import {AnalyticsToolsDecorator} from 'story-utils/decorators';
+
+import AbortionButton from './AbortionButton';
+
+export default {
+  title: 'Private API / Abortion button',
+  component: AbortionButton,
+  decorators: [AnalyticsToolsDecorator],
+};
+
+export const Authenticated = {
+  args: {
+    isAuthenticated: true,
+  },
+  play: async ({canvasElement}) => {
+    const canvas = within(canvasElement);
+
+    await waitFor(async () => {
+      const abortButton = await canvas.queryByRole('button', {name: 'Uitloggen'});
```

Comment:
Why not `getByRole`, if we know it will be there?

Reply:
I always hesitate when I want to test if a component is present on whether to use 'get' and then let it error immediately in case it can't find it, or use 'query' and let it error on the expect. I find the latter clearer in terms of 'what is this test testing' :thinking: what do you think?

Reply:
I don't have any preference, I find the test API not that user friendly anyway in the first place :sweat_smile: